src/Spout/Writer/ODS/Manager/Style/StyleManager.php

```diff
@@ -319,6 +336,22 @@ private function transformCellAlignment($cellAlignment)
         }
     }
 
+    /**
+     * Spec uses 'middle' rather than 'center'
+     * http://docs.oasis-open.org/office/v1.2/os/OpenDocument-v1.2-os-part1.html#__RefHeading__1420236_253892949
+     *
+     * @param string $cellAlignment
+     *
+     * @return string
+     */
+    private function transformCellVerticalAlignment($cellVerticalAlignment)
+    {
+        switch ($cellVerticalAlignment) {
```

Comment:
```
return ($cellVerticalAlignment === CellVerticalAlignment::CENTER)
    ? 'middle'
    :  $cellVerticalAlignment;
```

Reply:
I've adjusted this code as you've suggested.